catroweb.yaml

```diff
@@ -694,6 +721,15 @@ components:
           - most_downloaded
           - example
           - scratch
+    RecommendedProjectType:
+      in: query
+      name: category
+      required: true
+      schema:
+        type: string
+        enum:
+          - similar
+          - specific
```

Comment:
We made some tiny changes so it would look like this:
```yaml
 RecommendedProjectType:
      in: query
      name: category
      required: true
      schema:
        type: string
        enum:
          - similar
          - also downloaded
      description: |
       * `similar` - TODO DESCRTIPTION !!!
       * `also downloaded` - TODO DESCRTIPTION!!!
```
as it is possible to describe the parameters in enums :> and makes it easier to understand. Please also write meaningful descriptions 👍